crates/re_viewport/src/viewport_blueprint_ui.rs

```diff
@@ -399,35 +402,50 @@ impl Viewport<'_, '_> {
             |ui| {
                 ui.style_mut().wrap = Some(false);
 
-                let add_space_view_item = |ui: &mut egui::Ui, space_view: SpaceViewBlueprint| {
-                    if ctx
-                        .re_ui
-                        .selectable_label_with_icon(
-                            ui,
-                            space_view.class(ctx.space_view_class_registry).icon(),
-                            if space_view.space_origin.is_root() {
-                                space_view.display_name.clone()
-                            } else {
-                                space_view.space_origin.to_string()
-                            },
-                            false,
-                        )
-                        .clicked()
-                    {
-                        ui.close_menu();
-                        ctx.selection_state()
-                            .set_selection(Item::SpaceView(space_view.id));
-
-                        let new_ids = self.blueprint.add_space_views(
-                            std::iter::once(space_view),
-                            ctx,
-                            None, //TODO(ab): maybe add to the currently selected container instead?
-                        );
-                        if let Some(new_id) = new_ids.first() {
-                            self.blueprint.focus_tab(*new_id);
+                let add_space_view_item =
+                    |ui: &mut egui::Ui, space_view: SpaceViewBlueprint, empty: bool| {
+                        let label = if empty {
+                            format!(
+                                "Empty {} view",
+                                space_view
+                                    .class(ctx.space_view_class_registry)
+                                    .display_name()
+                            )
+                        } else {
+                            format!(
+                                "{} view of {}",
+                                space_view
+                                    .class(ctx.space_view_class_registry)
+                                    .display_name(),
+                                space_view.space_origin
+                            )
+                        };
+
+                        if ctx
+                            .re_ui
+                            .selectable_label_with_icon(
+                                ui,
+                                space_view.class(ctx.space_view_class_registry).icon(),
+                                label,
+                                false,
+                                false,
```

Comment:
This is the problem with boolean parameters. Is `false, false` the correct arguments here? As a reviewer on GitHub, I have no idea.